0.1.6_upgrade.sql

```diff
@@ -0,0 +1,13 @@
+INSERT INTO packages (pkg_name) VALUES (SELECT DISTINCT pkg_name FROM files);
+
+INSERT INTO releases (pkg_name, tarball, tar_sum) VALUES (
+    SELECT DISTINCT pkg_name, tar_file, tar_sum FROM files
```

Comment:
I don't think there is a ``tar_file`` column here to select.  It seems like it was renamed to ``tarball`` in 165ff5e04